erts/preloaded/src/socket.erl

```diff
@@ -3982,8 +3980,8 @@ invalid_type(Type) ->
       Type  :: term(),
       Proto :: term().
 
-invalid_protocol(Type, Proto) ->
-    error({invalid_protocol, {Type, Proto}}).
+invalid_protocol(Proto) ->
+    error({invalid_protocol, Proto}).
```

Comment:
The spec must be changed too

Reply:
Which specs?

Reply:
The spec for this function: `invalid_protocol(Proto)`. The spec 4 lines up is for `invalid_protocol(Type, Proto)` which is a now nonexistent function.